Codegen/src/Generators/DefaultGenerator.cs

```diff
@@ -877,13 +880,18 @@ private static void AdvanceToNextStateAndAdjustTokenProperties(TokensLine tempTo
                         base.Layout);
 
                     int startIndex = 0;
-                    if (tempTokensLine.Type == CobolTextLineType.Debug && cel.ScanState.WithDebuggingMode && base.Layout == ColumnsLayout.CobolReferenceFormat)
+                    if (tempTokensLine.Type == CobolTextLineType.Debug && cel.ScanState.WithDebuggingMode)
                     {//Handling special debugging mode.
-                        System.Diagnostics.Debug.Assert(line.Length >= 7 && (line[6] == 'D' || line[6] == 'd'));
-                        if (line.Length >= 7 && (line[6] == 'D' || line[6] == 'd'))
+                        int minIndCol = base.Layout == ColumnsLayout.CobolReferenceFormat ? 6 : 0;
+                        int maxIndCol = base.Layout == ColumnsLayout.CobolReferenceFormat ? 7 : 2;
```

Comment:
Is the name `maxIndCol` correct ? It looks like, it's rather the "first meaningful char"

Reply:
yes max indicator column